### PR TITLE
more verbose data access with download links

### PR DESCRIPTION
**Pull request by smpetersen**
Added direct urls for data download and created separate headings for data access via S3 and URL with more verbose bucket paths.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +116,7 @@ The data is also available in Apache Parquet format. The available fields are de
 
 ### R Package
```
Comment (katiejolly-ookla) [inline README.md:117]: Now that there's a longer description of how to download the data, it seems like the R Package section might below below "Download via URL"

File: README.md [hunk 2]
```diff
@@ -49,15 +49,42 @@ Data is subject to be reaggregated regularly in order to honor Data Subject Acce
 
 ## Data Access and Formats
 
-Data are provided in both Shapefile format as well as Apache Parquet, and can be accessed on Amazon Web Services (AWS) object storage service, S3, at the following paths:
+Data are provided in both Shapefile format as well as Apache Parquet.
 
-* `s3://ookla-open-data/parquet/performance/`
-* `s3://ookla-open-data/shapefiles/performance/`
+### Access via S3
```
Comment (dled) [inline README.md:54]: ### Access via S3

The fixed and mobile performance datasets can be found at the following paths on the  `s3://ookla-open-data` bucket, organized by

1. file format (`shapefiles` or `parquet`)
2. service type (`fixed` or `mobile`)
3. year (`2020`)
4. quarter (`1` corresponds to the period starting `2020-01-01`)

```bash
export FORMAT='shapefiles' # (shapefiles|parquet)
export TYPE='fixed'        # (fixed|mobile)
export YYYY='2020'         # 2020
export Q='1'               # 1,2,3 (to date)

aws s3 cp s3://ookla-open-data/${FORMAT}/performance/type=${TYPE}/year=${YYYY}/quarter=${Q}/ . \
--recursive \
--no-sign-request
```

In the example above, the `--recursive` option downloads the compressed shapefiles corresponding to the year, quarter, and service type of the S3 endpoints we passed to the S3 copy command. In this case, `2020-01-01_performance_fixed_tiles.zip`.

To download the mobile and fixed datasets in parquet format for 2020Q3:

``` bash
# Mobile 2020 Q3
aws s3 cp s3://ookla-open-data/parquet/performance/type=mobile/year=2020/quarter=3/2020-07-01_performance_mobile_tiles.parquet

# Fixed 2020 Q3
aws s3 cp s3://ookla-open-data/parquet/performance/type=fixed/year=2020/quarter=3/2020-07-01_performance_fixed_tiles.parquet
```

File: README.md (hunk 1 above)
Comment (smpetersen) [reply README.md:117]: good idea!

**Review by dled — APPROVED** [review]
will open a PR for explicit `awscli` access.

File: README.md (hunk 2 above)
Comment (smpetersen) [reply README.md]: nice! I'll incorporate the explicit keyname definitions in this pr.

Comment (smpetersen) [reply README.md]: And I really like the idea of a bash example for using the aws cli. Can you take the lead on a separate pr for that?

File: README.md (hunk 1 above)
Comment (smpetersen) [reply README.md:120]: I ended up separating out and moving the data formats section above the data access section.

**Review by dled — APPROVED** [review]

**Review by katiejolly-ookla — APPROVED** [review]
looks great!

File: README.md [hunk 3]
```diff
@@ -87,11 +78,50 @@ These files serve the following purpose:
...
+
+The performance datasets can be accessed on Amazon Web Services (AWS) object storage service, S3, at the s3://ookla-open-data bucket, organized by the key names:
+
+1. file format (shapefiles or parquet)
+2. service type (fixed or mobile)
+3. year (2020)
+4. quarter (1 corresponds to the period starting 2020-01-01)
+
+Using the key names for the file format, type, year, quarter, and data layer, one can specify the desired dataset. For example, to access all of the files for fixed and mobile service types for the first quarter of the year 2020, one would use the following buckets:
```
Comment (jj-ookla) [inline README.md:92]: > one would use the following buckets:

buckets -> S3 URIs

**Review by jj-ookla — COMMENTED** [review]
Looks great! 

Sorry I'm late to the party but only suggestion is to update verbiage re buckets / S3 URI.

**Comment by dled** [comment]
@jj-ookla Will add to this next PR for https://github.com/teamookla/ookla-open-data/tree/add_awscli_docs

**Comment by smpetersen** [comment]
thanks, Jeremy and Dylan!

**Pull request merged**